roles/ceph-validate/tasks/check_system.yml

```diff
@@ -29,12 +29,13 @@
 - name: red hat based systems tasks
   when:
     - ceph_repository == 'rhcs'
-    - ansible_distribution == 'Red Hat Enterprise Linux'
+    - ansible_distribution == 'RedHat'
   block:
     - name: fail on unsupported distribution for red hat ceph storage
       fail:
-        msg: "Distribution not supported {{ ansible_distribution_version }} by Red Hat Ceph Storage, only RHEL >= 7.3"
-      when: ansible_distribution_version | version_compare('7.3', '<')
+        msg: "Distribution not supported {{ ansible_distribution_version }} by Red Hat Ceph Storage, only RHEL >= 8.1 or >= 7.7"
```

Comment:
the message doesn't look really clear:

eg: 8.0 is >= 7.7 but we know this is not supported